app/javascript/lib/scp-api.js

```diff
@@ -120,6 +120,67 @@ export function setMockOrigin(origin) {
   mockOrigin = origin
 }
 
+/** Constructs and encodes a URL parameter, if it has a value */
+function encodeParam(param, value, isFirst=false) {
```

Comment:
I like that you pulled this function out, but I think it's duplicative of functionality in the query-string module we're already using elsewhere. https://www.npmjs.com/package/query-string

Reply:
Good call!  [Done](b24e07f).